test/message.test.js

```diff
@@ -43,17 +44,17 @@ describe('message', () => {
   });
 
   it('avatar should be set with provided user', () => {
-    message.user = user;
+    setUser(message);
     expect(avatar.getAttribute('name')).to.be.equal('Joan Doe');
     expect(avatar.getAttribute('abbr')).to.be.equal('JD');
     expect(avatar.getAttribute('img')).to.be.equal('/test/visual/avatars/avatar.jpg');
-    expect(avatar.getAttribute('title')).to.be.equal('Joan Doe (JD)');
+    expect(avatar.getAttribute('title')).to.be.equal('Joan Doe');
```

Comment:
Why has this changed?

Reply:
Avatar behaves differently when you put the properties declaratively vs programmatically. I consider it a bug. I filed a ticket to avatar. https://github.com/vaadin/vaadin-avatar/issues/106

Reply:
Note: web-padawan commented, rightfully, that setting the title is an internal detail of avatar and shouldn't be tested in message. I removed the row in question.